misc/security-and-privacy.md

```diff
@@ -0,0 +1,96 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+This questionnaire has [moved](https://w3ctag.github.io/security-questionnaire/).
+
+# Security and Privacy Self-review for IMSC-HRM
+
+This is an edited version of the security and privacy questionnaire template at
+https://raw.githubusercontent.com/w3ctag/security-questionnaire/main/questionnaire.markdown as linked from
+[Self-Review Questionnaire: Security and Privacy](https://www.w3.org/TR/security-privacy-questionnaire/)
+
+> 01.  What information might this feature expose to Web sites or other parties,
+>      and for what purposes is that exposure necessary?
+
+None. The IMSC-HRM is a static-analysis algorithm of IMSC documents.
+Executing the algorithm requires no requests to any origins.
```

Comment:
Is it worth explicitly noting that "This specification does not define any protocol or interface for obtaining such a resource, and it does not define any interface for exposing the results of the analysis." as mentioned at https://github.com/w3c/imsc-hrm/issues/19#issue-1073404996

Reply:
Yes will do.

Reply:
Done in e971621.